src/build/__init__.py

```diff
@@ -332,14 +339,25 @@ def _call_backend(self, callback, outdir, config_settings=None, **kwargs):
         else:
             os.mkdir(outdir)
 
+        with self._handle_backend(callable.__name__):
+            basename = callback(outdir, config_settings, **kwargs)  # type: str
+            return os.path.join(outdir, basename)
```

Comment:
I think the return line might be de-indented :thinking:

Reply:
Eh, doesn't matter anyway, the context manager will run perfectly fine and close after the return.

Reply:
Nevermind, we want to prevent capturing exceptions from `os.path.join` 🤦

Reply:
Fixed.